### ISSUE TITLE
DT and Shiny Render text not compatible

### ISSUE THREAD

**Issue by valentinlefranc** [rstudio/DT#376]
Hello,

I managed to figure out that the update DT package 0.2 instead of 0.1 create some problems in my shiny apps. Indeed when I add some DT::DataTableOutput the rendertex of the same page disappear whereas it is not the case with DT version 0.1 ...
Can it be fixed ? ;)

Thanks!
Valentin

**Comment by yihui** [comment]
Could you provide a minimal, self-contained, and reproducible example? Please also include your session info:

```r
devtools::session_info('DT')
```

**Comment by valentinlefranc** [comment]
Hi, thanks for your reply. It is quit hard for me to repoduce an example (sorry) the problem occured in a complex shiny app. (I have tried in a dummy one but the problem do no occur). In addition some DT create the problem other don't (...). The way I have coded the display of the DT is strictly the same. The only way I found to display my render text is to comment the problematic DT or downgrade to DT 0.1 ....

Here is the results of devtools::session_info('DT')


Session info -------------------------------------------------------------------------------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.3.1 (2016-06-21)
 system   x86_64, mingw32             
 ui       RStudio (0.99.893)          
 language (EN)                        
 collate  French_France.1252          
 tz       Europe/Paris                
 date     2016-12-01                  

Packages -----------------------------------------------------------------------------------------------------------------------------------------------------------------------
 package     * version date       source        
 digest        0.6.10  2016-08-02 CRAN (R 3.3.1)
 DT            0.2     2016-08-09 CRAN (R 3.3.1)
 htmltools     0.3.5   2016-03-21 CRAN (R 3.3.1)
 htmlwidgets   0.7     2016-08-02 CRAN (R 3.3.1)
 jsonlite      1.1     2016-09-14 CRAN (R 3.3.1)
 magrittr      1.5     2014-11-22 CRAN (R 3.3.1)
 Rcpp          0.12.7  2016-09-05 CRAN (R 3.3.1)
 yaml          2.1.13  2014-06-12 CRAN (R 3.3.1)
